source/features/mobile-tabs.css

```
@@ -67,7 +67,11 @@
 		justify-self: end;
 	}
 
-	:root .UnderlineNav-item .Counter {
+	:root .UnderlineNav-item span {
```

```suggestion
	/* Tab label on on profiles */
	:root .UnderlineNav-item span:not([class]) {	
```

I'd also fix the extra padding and reduce the font size